ci/e2e/e2e.go

```diff
@@ -43,5 +43,9 @@ func upDependencies() error {
 }
 
 func upApp() error {
-	return sh.RunV("docker-compose", "-f", e2e.DockerFile, "up", "--build", "-d", "discovery")
+	err := sh.RunV("docker-compose", "-f", e2e.DockerFile, "up", "--build", "-d", "discovery")
```

Comment:
any reason to not do this in the same command?

Reply:
No. Didn't know it accept multiple services

Reply:
I think it does

Reply:
Correct. I checked. Will do this change in next pr